src/Nest/Indices/IndexManagement/RolloverIndex/RolloverConditions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nest
+{
+	[JsonObject(MemberSerialization.OptIn)]
+	[JsonConverter(typeof(ReadAsTypeJsonConverter<RolloverConditions>))]
+	public interface IRolloverConditions
+	{
+		[JsonProperty("max_age")]
+		Time MaxAge { get; set; }
+
+		[JsonProperty("max_docs")]
+		int MaxDocs { get; set; }
```

Comment:
[this is defined as a `long` in Elasticsearch](https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/action/admin/indices/rollover/MaxDocsCondition.java#L31)